app/src/main/java/org/simple/clinic/monthlyReports/questionnaire/entry/QuestionnaireEntryModel.kt

```diff
@@ -0,0 +1,30 @@
+package org.simple.clinic.monthlyReports.questionnaire.entry
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+import org.simple.clinic.facility.Facility
+import org.simple.clinic.monthlyReports.questionnaire.Questionnaire
+
+@Parcelize
+data class QuestionnaireEntryModel(
+    val facility: Facility?,
```

Comment:
Why does a QuestionnaireEntry have a facility? Facility field will be possessed only by QuestionnaireResponse

Is QuestionnaireEntry synonymous to QuestionnaireResponse? If so, shall we name it as response?

Reply:
Also, facility isn't optional. It is a required field. Thought I'd add a note since I saw a question at the end `Facility?`.